docs/admin/authorization.md

```diff
@@ -204,7 +204,7 @@ As of 1.3 RBAC mode is in alpha and considered experimental.
 
 To use RBAC, you must both enable the authorization module with `--authorization-mode=RBAC`,
 and [enable the API version](
-cluster-management.md/#Turn-on-or-off-an-API-version-for-your-cluster),
+/docs/admin/cluster-management/#turn-on-or-off-an-api-version-for-your-cluster),
```

Comment:
Stylistically it's usually easier and less confusing to link everything relative to root. I'm fine with this.